12LessonTwelve/homework/css/style.css

```diff
@@ -0,0 +1,90 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css">
+body {
+  background: ;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+header {
+  font-family: 'Lily Script One', cursive;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  font-size: 1em;
+  font-weight: bold;
+}
+article > div {
+  font-family: sans-serif;
+  text-align: center;
+  border: 1px solid #a5a5aa;
+  margin: 0 3%;
+}
+* Media Queries
+ **/
+@media screen and (min-width: 320px) {
+  .container::before {
+    content: "320px";
+    color: green;
+    position: absolute;
+    top: 0;
+    left: 0;
+    font-weight: 700;
+    margin: 5px;
+    font-size: 2em;
+  }
+}
+@media screen and (min-width: 480px) {
+  .container::before {
+    content: "480px";
+    color: maroon;
+    }
+}
+  article {
+    margin-left: 9%;
+    margin-right: 9%;
+}
+@media screen and (min-width: 620px) {
+  .container::before {
+    content: "620px";
+    color: orange;
+    }
+}
+  section {
+    display: flex;
+    flex-flow: wrap;
+}
+  article {
+    margin: 0 auto;
+    margin-bottom: 60px;
+    flex-basis: 300px;
+}
+  .price {
+    width: 70px;
+}
+@media screen and (min-width: 810px) {
+  .container::before {
+    content: "810px";
+    color: black;
+  }
+}
+  article {
+    flex-basis: 350px;
```

Comment:
Try using a responsive unit like %. Since you want 4 items side by side, 25% should about do it.